system_tests/redis_test.go

```diff
@@ -1,13 +1,10 @@
 // Copyright 2021-2022, Offchain Labs, Inc.
 // For license information, see https://github.com/nitro/blob/master/LICENSE
 
-//go:build redistest
-// +build redistest
-
 package arbtest
 
 import "testing"
 
 func TestRedisBatchPosterParallel(t *testing.T) {
-	TestBatchPosterParallel(t)
+	testBatchPosterParallel(t, true)
```

Comment:
Let's just put this in `batch_poster_test.go` now that it's not behind a build tag

Reply:
done.